File: prob1.py
# Time Complexity : O(nlogn)
# Space Complexity :O(1)
# Passed on Leetcode: yes

class Solution:
    def arrayPairSum(self, nums: List[int]) -> int:
        nums.sort()
        s = 0
        for i in range(0, len(nums), 2):
            s += nums[i]

        return s

File: prob1.java
// Time Complexity : O(nlogn)
// Space Complexity : O(1)
// Did this code successfully run on Leetcode : Yes

class Solution {
    public int arrayPairSum(int[] nums) {
        Arrays.sort(nums);
        int sum=0;
        for(int i=0;i<nums.length;i+=2)
        {
          sum+=nums[i];
        }
        return sum;
    }
}